<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partspro</class>
 <widget class="QMainWindow" name="partspro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>partspro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#partspro
{
background-color: rgb(234, 234, 234);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="maingroupbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>611</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#maingroupbox{
border: 1px black;
border-radius: 20px;
background: white;
;
border-color:black;
;

font: 12pt &quot;Calibri&quot;;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>591</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView#tableView

{

 
	font: 12pt &quot;Calibri&quot;;
	background-color: rgb(225, 223, 224);
	border-radius: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>110</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(225, 223, 224);
border-radius: 10px</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Entry Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EdatelineEdit_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="OdatelineEdit_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Out Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="porderlineEdit_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Purchase order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="todaypushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover
{
background-color: rgb(0, 85, 255);

}
QPushButton:disabled {
	
	background-color:rgb(225, 223, 224);
}
QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	background-color: rgb(0, 170, 255);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>Today</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="showRecordspushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>510</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Calibri&quot;;
color: black;

</string>
     </property>
     <property name="text">
      <string>Show Records</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditpushButton_2">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>570</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
 
	background-color: rgb(75, 150, 0);

}
QPushButton:disabled {
	
	background-color: rgb(198, 198, 198);
}
QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	 
	background-color: rgb(94, 189, 0);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearpushButton_3">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>570</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
 
	background-color:rgb(200, 180, 22);

}
QPushButton:disabled {
	
	background-color: rgb(198, 198, 198);
}
QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	 
	background-color:rgb(220, 220, 0);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleterpushButton_7">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>570</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
 
	background-color: rgb(127, 10, 0);

}
QPushButton:disabled {
	
	background-color:rgb(225, 223, 224);
}
QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	 
	background-color: rgb(226, 20, 6);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seeinventorypushButton_4">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>60</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*font: 12pt &quot;Calibri&quot;;
color: white;

background-color: rgb(0, 170, 255);
*/
QPushButton:hover
{
background-color: rgb(0, 85, 255);

}

QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	background-color: rgb(0, 170, 255);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Go to Inventory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchlineEdit_5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="logolabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/logo/mit.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>230</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(225, 223, 224);
border-radius: 10px</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="pnolineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="PNametextEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Part No.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Part Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Quantity assigned</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quantitylineEdit_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>380</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(225, 223, 224);
border-radius: 10px</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="RFNdatelineEdit_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Requisiton Form No.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="VRNdatelineEdit_5">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>VRN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CNameineEdit_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:white;
font: 12pt &quot;Calibri&quot;;
padding: 1px;
border-style: solid;
border: 1px solid white;
border-radius: 8px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot; ;
color: rgb(120, 120, 120);
padding-bottom: 0;</string>
      </property>
      <property name="text">
       <string>Customer's Name</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Parts Record Management</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
